tests/login.test.js

```diff
@@ -19,5 +19,4 @@ describe('log in to the admin page', () => {
       .then(() => browser.getText('h1.page-header'))
       .then(header => expect(header).to.equal('Overview'));
   });
-
 });
```

Comment:
removed this test as we are already using it a lot throughout our tests 

readonly is tested in users

logout is used throughout

will revisit once I get it set up for realm login